src/test/java/com/griddynamics/cloud/learning/service/ArticleServiceTest.java

```diff
@@ -0,0 +1,63 @@
+package com.griddynamics.cloud.learning.service;
+
+import com.griddynamics.cloud.learning.dao.domain.Article;
+import com.griddynamics.cloud.learning.dao.domain.PaginatedResult;
+import com.griddynamics.cloud.learning.web.dto.*;
+import org.junit.Test;
+
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ArticleServiceTest {
+
+    private ArticleService service = new ArticleService();
```

Comment:
We have to instantiate service, repo and all other components every time.
How could we make it a singleton?